stories/demos/components/HeaderRenderers/DraggableHeaderRenderer.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
```

Comment:
I'm just not sure about this file. What do you guys think if we provide this in the DataGrid and export this as a default `DraggableHeaderRenderer`?

Reply:
I would prefer to keep the dependencies to a minimum and provide a flexible API so users can write their own implementations. Composition is always more maintainable than adding extra props

Reply:
Yes we have to maintain the feature + dependency on other libs.
That way users can use whatever implementation they want to header dragging.